content/graphql/README.md

```diff
@@ -0,0 +1,10 @@
+# GraphQL
+
+The `/content/graphql` directory is where the GitHub GraphQL API docs live!
+
+* The `/content/graphql/guides` and `/content/graphql/overview` directories contain regular articles. These are human-editable.
+* The `/content/graphql/reference` directory contains a skeleton structure for the GraphQL API reference. However, most GraphQL API content inserted using `include`s.
```

Comment:
I'm not sure `skeleton structure` is very universal/translatable. What do you think about:

```suggestion
* The `/content/graphql/reference` directory contains an article for each GraphQL data type used in the GitHub API. Most of the content in this directory is rendered using the `include` tag. 
```